src/test/java/org/folio/dao/DataImportQueueDaoTest.java

```diff
@@ -37,35 +41,54 @@ public class DataImportQueueDaoTest  {
 
   @InjectMocks
   DataImportQueueItemDao queueItemDaoImpl = new DataImportQueueItemDaoImpl();
-  
 
   UUID storedItemUUID;
+
   @Before
   public void setUp() {
-    MockitoAnnotations.initMocks(this);
+    MockitoAnnotations.openMocks(this);
     when(postgresClientFactory.getInstance()).thenReturn(postgresClient);
   }
-  @Test 
+
+  @Test
   public void shouldAddQueueItem(TestContext context) {
     // given
-    when(postgresClient.execute(anyString(), any(Tuple.class)))
-      .thenReturn(Future.succeededFuture(new LocalRowSet(1)));
+    doAnswer((InvocationOnMock invocation) -> {
+        Promise<RowSet<Row>> promise = invocation.getArgument(2);
```

Comment:
Have to do some funkiness here to cast the argument to a `Promise`, since `Promise` is parameterized.  Normally, we could just do `invocation.getArgument(2, Promise.class).complete(...)` but for this we need that extra variable